ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/nested-virtualization/deprovision/cucushift-installer-rehearse-gcp-ipi-nested-virtualization-deprovision-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-nested-virtualization-deprovision
+  steps: 
+  - ref: gather-installer-boostrap-logs-in-bastion
```

Comment:
This step is only used in private cluster install, so we do not need this step here.